groups/restrictions.yaml

```diff
@@ -336,6 +336,7 @@ restrictions:
       - "^wg-lts-leads@kubernetes.io$"
   - path: "wg-device-management/groups.yaml"
     allowedGroups:
+      - "^k8s-infra-staging-dra-driver-cpu@kubernetes.io$"
       - "^k8s-infra-staging-dra-example-driver@kubernetes.io$"
```

Comment:
workgroups cannot own code / infrastructure, just that needed to organize the group, these shouldn't be here

these projects should be under a SIG, it looks like SIG Node sponsors this project?

Reply:
https://github.com/kubernetes/k8s.io/pull/8340